docs/contributing.rst

```diff
@@ -132,29 +133,36 @@ Commit changes to git::
 
     git commit -a -m "Prepare release <new-version>"
 
-Test the new version with Tox_::
-
-    tox -r
+Bump project's version in ``resolwe/__about__.py`` file to the next
```

Comment:
Please wrap the lines at 80 (79) characters, not before.

Applies for all changes.